shield_advanced.tf

```diff
@@ -11,13 +11,14 @@ module "shield_advanced_label" {
 resource "aws_fms_policy" "shield_advanced" {
   for_each = local.shield_advanced_policies
 
-  name                        = module.shield_advanced_label[each.key].id
-  delete_all_policy_resources = lookup(each.value, "delete_all_policy_resources", true)
-  exclude_resource_tags       = lookup(each.value, "exclude_resource_tags", false)
-  remediation_enabled         = lookup(each.value, "remediation_enabled", false)
-  resource_type_list          = lookup(each.value, "resource_type_list", null)
-  resource_type               = lookup(each.value, "resource_type", null)
-  resource_tags               = lookup(each.value, "resource_tags", null)
+  name                               = module.shield_advanced_label[each.key].id
+  delete_all_policy_resources        = lookup(each.value, "delete_all_policy_resources", true)
+  delete_unused_fm_managed_resources = lookup(each.value, "delete_unused_fm_managed_resources", false)
```

Comment:
please add this value to the description of the variable
https://github.com/cloudposse/terraform-aws-firewall-manager/blob/main/variables.tf#L137

Reply:
@milldr - Done.